tests/kernel/spinlock/src/main.c

```diff
@@ -137,6 +137,8 @@ ZTEST(spinlock, test_spinlock_bounce)
 		bounce_once(1234, false);
 	}
 
+	k_thread_abort(&cpu1_thread);
```

Comment:
Probably a good idea to do `k_thread_join()` to make sure the thread is no longer running.

Reply:
Waiting for termination is a guarantied part of abort's semantics.  Abort and join actually share the same implementation.